user-guide-chapters/zli/zli.adoc

```diff
@@ -3,26 +3,69 @@
 zli is a binary that implements a set of command line commands for interacting
 with the zot registry server.
 
-== Build zli from Source
+== How to Get zli
```

Comment:
lets move this to the getting started section at the top. Basically we should ask people to download these supporting tools as they download zot itself. In this chapter lets focus on how to use this tool. I would actually make zli the first chapter after getting started. In the getting started lets a section to do basic bringup in 2 mins workflow of a very simple zot server. Then go into zli chapter to show how to use this tool against the zot server. Then we add the server configuration chapter which will outline all configuration/deployment models and knobs/dials.